assets/builtin-policy.csv

```diff
@@ -14,6 +14,7 @@ p, role:readonly, projects, get, *, allow
 p, role:readonly, accounts, get, *, allow
 p, role:readonly, gpgkeys, get, *, allow
 p, role:readonly, logs, get, */*, allow
+p, role:readonly, exec, get, */*, allow
```

Comment:
@alexmt Is that intended? Giving exec to every readonly user looks somewhat dangerous to me?